processor/k8sprocessor/processor.go

```diff
@@ -27,8 +26,11 @@ import (
 )
 
 const (
-	k8sIPLabelName    string = "k8s.pod.ip"
-	clientIPLabelName string = "ip"
+	k8sIPLabelName     string = "k8s.pod.ip"
+	k8sPodUIDLabelName string = "k8s.pod.uid"
+	podUIDLabelName    string = "pod_uid"
+	clientIPLabelName  string = "ip"
+	hostnameLabelName  string = "host.hostname"
```

Comment:
open-telemetry/opentelemetry-specification#838 removed `host.hostname` from the spec in favor of `host.name`. I would suggest using `host.name`.

On a related note, some of these are available on the `go.opentelemetry.io/collector/translator/conventions` package as constant, it's better to use those if possible.